File: GSI QA testing tool/UI/UI_015_UnemploymentInsurance.cs
using GSI_QA_testing_tool.Utilities;
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GSI_QA_testing_tool.UI
{
    internal class UI_015_UnemploymentInsurance
    {
        public static void GoTo(IWebDriver driver)
        {


            if (Data._StopAt.Contains("Unemployment Insurance Compensation"))
            {
                CustomDialog customDialog = new CustomDialog("Make Selection", "Select pages you want to stop at, make changes and click ok before clicking the Next button");
                customDialog.ShowDialog();
            }




            if (Finder.FindIt(driver, "//label[@for='ctl00_Main_content_radFilingUI_0']"))
            {
                Debug.WriteLine("UnemploymentInsurance is On");
            }
            else
            {
                Debug.WriteLine("UnemploymentInsurance is Off");
                return;
            }




            Finder.ClickIt(driver, "//label[@for='ctl00_Main_content_radFilingUI_0']");
            Finder.ClickIt(driver, "//input[@id='ctl00_Main_content_btnNext']");

        }
    }
}

File: GSI QA testing tool/UI/UI_015_UnemploymentInsurance.vb
Imports OpenQA.Selenium
Imports System
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace GSI_QA_testing_tool.UI
    Friend Class UI_015_UnemploymentInsurance
        Public Shared Sub [GoTo](ByVal driver As IWebDriver, ByVal js As IJavaScriptExecutor)
            If Data._StopAt.Contains("Unemployment Insurance Compensation") Then
                Data.StopAtDialog()
            End If

            If Finder.FindItByCSS(driver, "#ctl00_Main_content_radFilingUI_0") Then
                Debug.WriteLine("UnemploymentInsurance is On")
            Else
                Debug.WriteLine("UnemploymentInsurance is Off")
                Return
            End If

            Finder.JSClickIt(driver, "#ctl00_Main_content_radFilingUI_0", js)
            Finder.JSClickIt(driver, "#ctl00_Main_content_btnNext", js)
        End Sub
    End Class
End Namespace
